components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -208,6 +210,7 @@ export class ClusterService implements IClusterServiceServer {
     }
 
     public deregister(call: grpc.ServerUnaryCall<DeregisterRequest, DeregisterResponse>, callback: grpc.sendUnaryData<DeregisterResponse>) {
+        log.info(getUserAgentAndClientName(call), "requested clusters.deregister")
```

Comment:
the order of the arguments should be reversed: `log.info("requested clusters.deregister", getUserAgentAndClientName(call))`

Reply:
done